src/soundbase.cpp

```diff
@@ -184,14 +181,14 @@ QString CSoundBase::SetDev ( const QString strDevName )
             // to be able to access the ASIO driver setup for changing, e.g., the sample rate, we
             // offer the user under Windows that we open the driver setups of all registered
             // ASIO drivers
-            sErrorMessage = sErrorMessage + "<br/>" + tr ( "Do you want to open the ASIO driver setups?" );
+            sErrorMessage = sErrorMessage + "<br/>" + tr ( "Do you want to open the ASIO driver setup to change driver settings?" );
```

Comment:
Code consistency - use the `+=` operator to append to a QString.

Reply:
Fixed